eskip/eskip.go

```diff
@@ -44,13 +53,28 @@ type parsedRoute struct {
 	backend  string
 }
 
+// A CustomPredicate object represents a parsed, in-memory, route matching predicate
+// that is defined by extensions.
+type CustomPredicate struct {
```

Comment:
Why call it `CustomPredicate`? What would be the drawback of only having `Predicate` and implementing the default ones exactly like those (only that they are bundled)?